app/src/main/java/org/p2p/wallet/debug/featuretoggles/FeatureTogglesPresenter.kt

```diff
@@ -0,0 +1,41 @@
+package org.p2p.wallet.debug.featuretoggles
+
+import androidx.annotation.IdRes
+import kotlinx.coroutines.launch
+import org.p2p.wallet.R
+import org.p2p.wallet.common.AppFeatureFlags
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.settings.model.SettingsRow
+
+class FeatureTogglesPresenter(
+    private val appFeatureFlags: AppFeatureFlags
+) : BasePresenter<FeatureTogglesContract.View>(), FeatureTogglesContract.Presenter {
+
+    override fun loadData() {
+        launch {
+            view?.showSettings(getToggles())
+        }
+    }
+
+    override fun onToggleCheckedListener(@IdRes toggleId: Int, toggleChecked: Boolean) {
+        when (toggleId) {
+            R.id.enable_dev_net -> appFeatureFlags.setIsDevnetEnabled(toggleChecked)
+            R.id.polling_enabled -> appFeatureFlags.setPollingEnabled(toggleChecked)
+        }
+    }
+
+    private fun getToggles(): List<SettingsRow> {
```

Comment:
getToggles returns SettingsRow
how bout 
```suggestion
    private fun getSettingsRows(): List<SettingsRow> {
```